src/visual/boot.js

```diff
@@ -0,0 +1,89 @@
+let topComment = 
+`/**
+* Initializes the DFA object. 
+* This file was automatically generated from src/visual/boot.js.
+*/\n`;
+
+const { exec } = require('child_process');
+
+
+
+var fs = require('fs');
+//Get the file that is being edited
+var myArgs = process.argv.slice(2);
+console.log('myArgs: ', myArgs);
+if(myArgs.length > 1 || myArgs.length == 0){
+    throw Error ('One file needs to be specified');
+}
+var sourceFile = myArgs[0];
+
+//Specify the path to the miking executable in this variable:
+
+let pathToMiking = "/home/calin/KTH/Miking/miking/build/boot";
```

Comment:
This should not be hard-coded to your user ,)

Reply:
Fixed with ~/.local/bin/miking, I couldn't get it to install under /usr/bin so this should work for the moment.

Reply:
It should be only `mi` without path. You should assume that you have set `mi` in your path (not the hardcoded path). Also, note that the name is changed to `mi` since yesterday.

Reply:
Fixed to call "mi"